def throw():
    raise RuntimeError('This is the error message')

throw()